Lib/pydoc.py

```diff
@@ -70,6 +70,7 @@ class or function within a module or module in a package.  If the
 import sysconfig
 import time
 import tokenize
+import typing
```

Comment:
Typing is considered as a slow import, not all modules are allowed to import it, since it may slow down the startup time. 

But, since there are multiple other heavy imports (some of them might already import `typing`), I don't think it is that significant.

Reply:
yeah, i couldn't think of a good way around the import, other than matching on `__class__.__name__`.